src/audit.js

```diff
@@ -18,7 +18,7 @@ const defaultReports = {
   directory: `${process.cwd()}/lighthouse`,
 };
 
-const VALID_BROWSERS = ['Chrome', 'Chromium', 'Canary'];
+const VALID_BROWSERS = ['Chrome', 'Chromium', 'Canary', 'Edge', 'Opera'];
```

Comment:
could you kindly add this information in the readme please?

Reply:
Sure

Reply:
@abhinaba-ghosh , I have added a comment on playwright-lighthouse supported browsers on LambdaTest [here](https://github.com/abhinaba-ghosh/playwright-lighthouse/pull/62/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R445).
Regarding adding in README, it will then become a Playwright documentation rather than the documentation related to this library. It will involve the documentation of the commands https://playwright.dev/docs/api/class-browsertype#browser-type-launch-server and https://playwright.dev/docs/api/class-browsertype#browser-type-connect, which is not be related to this library.

We have just added `Edge` and `Opera` so that users don't get blocked when they run tests on these browsers.